test/docgen/view/__snapshots__/markdown.test.ts.snap

```diff
@@ -6473,18 +6473,18 @@ construct tree needs to be combined to be used as a single dependable.
 \`\`\`go
 import "github.com/aws/constructs-go/constructs/v10"
 
-constructs.NewDependencyGroup(deps IDependable) DependencyGroup
+constructs.NewDependencyGroup(deps ...IDependable) DependencyGroup
 \`\`\`
 
 | **Name** | **Type** | **Description** |
 | --- | --- | --- |
-| <code><a href="/packages/constructs/v/10.0.78/api/DependencyGroup.Initializer.parameter.deps?lang=python">deps</a></code> | <code><a href="/packages/constructs/v/99.99.99/api/IDependable?lang=python">IDependable</a></code> | *No description.* |
+| <code><a href="/packages/constructs/v/10.0.78/api/DependencyGroup.Initializer.parameter.deps?lang=python">deps</a></code> | <code>...github.com/aws/constructs-go/constructs/v10.IDependable</code> | *No description.* |
 
 ---
 
 ##### \`deps\`<sup>Required</sup> <a name="deps" id="DependencyGroup.Initializer.parameter.deps"></a>
 
-- *Type:* <a href="/packages/constructs/v/99.99.99/api/IDependable?lang=python">IDependable</a>
```

Comment:
Hmm, this doesn't look right...somehow we are dropping the link. This should be:

```console
...<a href="/packages/constructs/v/99.99.99/api/IDependable?lang=python">IDependable</a>
```